### DISCUSSION TITLE
Alert created from REST API: Ability to review alert config from UI in read-only mode.

### DISCUSSION THREAD

**Discussion by harish422** [grafana/grafana#65646] [Alerting]
Hi Grafana Team,

I noticed for any alerts created from REST APIs "Alertingprovisioning" module, they can't be edited or alert config can't be viewed from UI. I understand why write access should be cut down in UI.
But as an enhancement, is it possible for atleast to make the alert config(queries etc) to be visible from UI , but just read-only ?
right now we(and other) are finding it a bit hard to follow the alert config from UI for alerts created from REST API, maintaining the alert, config can only be done from API.
Not fair for "Anyone/everyone" to call(GET) the api's to quickly review the alert config. might not be their cup of tea.
This all what we can see right now. But reviewing the alert config is important to us.

![Screen Shot 2023-03-30 at 11 48 54 AM](https://user-images.githubusercontent.com/23487589/228935936-c914c76a-21b5-426d-bf20-3412acc51600.png)

**Comment by konrad147** [comment]
Hey, provisioned alerts preview has been available since Grafana 9.4. What version do you use?
We added query details to the rule details page in [this](https://github.com/grafana/grafana/pull/59626) PR

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This feature looks like it was implemented, so I will close this discussion. Please feel free to open an issue if there is something else you would like. Thank you!
